frontend/src/components/pages/acls/UserDetails.tsx

```diff
@@ -82,273 +85,149 @@ class UserDetailsPage extends PageComponent<{ userName: string }> {
 
     return (
       <PageContent>
+        <div className="flex justify-between">
+          <div>
+            <h3>Permissions</h3>
+            <p className="text-sm text-gray-600">The following permissions are assigned to this principal.</p>
+          </div>
+          <div className="flex gap-3">
+            {Features.rolesApi && (
+              <Button variant="outline" onClick={() => (this.isChangeRolesModalOpen = true)} disabled={!canEdit}>
+                Change roles
```

Comment:
Should this be Assign roles, instead of Change roles?

Reply:
Is there another verb? You can add or remove roles, i'll use that suggestion for now